### ISSUE TITLE
Individual servers for parallel testing clones

### ISSUE THREAD

**Issue by timominous** [envoy/Ambassador#56]
I'm not sure if I have missed the documentation for this but is it possible to run separate instances of `DefaultHTTPServer` on the individual clones while parallel testing?

**Comment by ilyapuchka** [comment]
Yes, just need to start it on different ports.

**Comment by timominous** [comment]
Yeah this is what I added handling for in my code. Just to confirm, is this strictly error code `48`?
